From 39222de1f05814d8c12d2f47fa641978777821b1 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 18 Jan 2024 12:07:31 +0100
Subject: [PATCH 0564/1695] plat-stm32mp2: conf: enable TA PKCS11

Default enable CFG_PKCS11_TA for STM32MP2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git b29f3e2ff2005f535a048f4f626358f798c68c00 plat-stm32mp2: conf: enable TA PKCS11]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: Ifc882ec404989d499da65e9e752e9eaea16fb69a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/352418
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 37c3958a9..37ebdf7c4 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -175,6 +175,9 @@ endif
 CFG_DRIVERS_NVMEM ?= y
 CFG_STM32_TAMP_NVRAM ?= y
 
+# Default enable PKCS11 TA to have PKCS11 tests built in OP-TEE Test
+CFG_PKCS11_TA ?= y
+
 # Provisioning support for BSEC shadow OTP is dedicated to insecure development
 # configuration only.
 CFG_STM32MP_PROVISIONING ?= y
-- 
2.47.2

